### PR TITLE
Add FY22 Q2 roadmap to Extensibility goals page

### PR DESCRIPTION
**Pull request by jeanduplessis**
@Joelkw @muratsu please feel free to edit and merge when the last one approves

### CODE REVIEW

**Comment by muratsu** [comment]
LGTM

**Comment by Joelkw** [comment]
(Just noting explicitly that I have seen this + this is on my list for this week, and I should be able to get to it Thursday if not Wednesday sometime.)

**Comment by Joelkw** [comment]
@jeanduplessis @muratsu I went ahead and mad a couple larger changes: 
1. I added "how we'll measure success" sections to each goal
2. I reworded the first few lines of the page to seem (IMO) less "what we can't do" and more "what we can do" 
3. I modified the roadmap section to include goal area and by FY-long, because I figured it fit with our aim of the page for "describe this team's work to another teammate or prospective candidate who is curious." Otherwise, some goals didn't seem to be on the roadmap (since they aren't in Q2). I didn't add absolutely everything, given some of these goals (like the product discovery one) will lead to others. 

Otherwise approving these changes / feel free to merge when you've approved. I will plan to do the same for code insights goals pages next. (And then circle back to updating completed goals for last quarter for both teams).

File: handbook/engineering/developer-insights/extensibility/goals.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Extensibility Team Goals and Roadmap
 
-The Extensibility team, due to missing dedicated PM, design and additional engineering support, will not be following a traditional product roadmap in FY22 and instead will focuses on high impact projects aligned with the company's OKRs.
+The Extensibility team focuses on high impact projects aligned with the company's OKRs. 
+
+As Sourcegraph continues grows, and the team is able to bring on a dedicated product manager, designer, and additional engineers, the team's roadmap will adopt a more traditional product development trajectory.
```
Comment (Joelkw) [inline handbook/engineering/developer-insights/extensibility/goals.md:5]:
```suggestion
As Sourcegraph continues to grow and the team is able to bring on a dedicated product manager, designer, and additional engineers, the team's roadmap will adopt a more traditional product development trajectory. 
```
```suggestion
As Sourcegraph continues to grow and the team is able to bring on a dedicated product manager, designer, and additional engineers, the team's roadmap will adopt a more traditional product development trajectory. 
```

File: handbook/engineering/developer-insights/extensibility/goals.md [hunk 2]
```diff
@@ -9,21 +12,43 @@ See also our [completed goals](goals_completed.md).
...
 
-We want – and our users want – our users to build custom extensions, but there are a number of blockers that keep other folks from making extensions. These include documentation issues, the extension registry's appearance sending mixed or incorrect messages about what extensions are and how actively they're maintained, the ability of a user to use many Sourcegraph extensions at once, and actually proving out the third party partnership/developer model for any third-party extension.
+We want – and our users want – our users to build custom extensions, but there are a number of blockers preventing this. These include documentation issues, the extension registry's appearance sending mixed or incorrect messages about what extensions are and how actively they're maintained, the ability of a user to use many Sourcegraph extensions at once, and validating the impact of a the third party partnership/developer model for major third-party extension.
```
Comment (Joelkw) [inline handbook/engineering/developer-insights/extensibility/goals.md:15]:
```suggestion
We want – and our users want – our users to build custom extensions, but there are a number of blockers preventing this. These include documentation issues, the extension registry's appearance sending mixed or incorrect messages about what extensions are and how actively they're maintained, the ability of a user to use many Sourcegraph extensions at once, and validating the impact of a the third party partnership/developer model for major third-party extensions.
```

**Comment by jeanduplessis** [comment]
@Joelkw as always you seem to have a way with words. Thanks for your updates.

**Pull request merged**